pkg/resource/scalable_target/testdata/scalable_target/v1alpha1/st_invalid_create_attempted.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: applicationautoscaling.services.k8s.aws/v1alpha1
```

Comment:
nit: full ResourceName in the filename might be better specially for smaller named resources such as scalable target.